src/main/java/graphql/nadel/ServiceExecutionParameters.java

```diff
@@ -66,6 +75,10 @@ public OperationDefinition getOperationDefinition() {
         return operationDefinition.getOperation();
     }
 
+    public Object getServiceContext() {
```

Comment:
Can we make this `<T> T getServiceContext()` - it makes it a smidge easier for the consumer since they KNOW what they put in

smidge
/smɪdʒ/
nounINFORMAL
another term for smidgen.
"a smidge over five foot two"

smidgen
/ˈsmɪdʒɪn/
nounINFORMAL
noun: smidgen; plural noun: smidgens; noun: smidgin; plural noun: smidgins; noun: smidgeon; plural noun: smidgeons
a small amount of something.

Reply:
changed